src/spares/Apps/oc_capture.lsl

```diff
@@ -326,6 +335,8 @@ default
                     if(Flag&1)g_iEnabled=TRUE;
                     if(Flag&2)g_iRisky=TRUE;
                     if(Flag&4)g_iCaptured=TRUE;
+                    if(Flag&8)g_iAutoRelease=TRUE;
+                    else g_iAutoRelease = FALSE;
```

Comment:
This line is probably not required. If the default state of this setting is false then this isn't needed :)